backend/src/api/PostController.ts

```diff
@@ -309,6 +316,7 @@ export default class PostController {
         }
         catch (err) {
             this.logger.error('Comment create failed', { error: err, user_id: userId, format, content, post_id: postId });
+            this.logger.error(error);
```

Comment:
is this needed? `error` is not defined anywhere

Reply:
It's a typo. good catch.